.circleci/config.yml

```diff
@@ -78,13 +78,13 @@ commands:
               fi
             fi
 
-  check_release_changelog_update_branch:
-    description: "Skip job for release changelog update PRs"
+  skip_for_low_risk_prs:
```

Comment:
@U8NWXD Are we expecting to move this to a separate action file and re-use it in all the workflow? (OR is it going to be a part of other project)

Reply:
This is in the Circle CI config, so I don't think action files are relevant here. Regarding GitHub Actions, do environment variables set inside an action persist in the workflow beyond that action? That's required for this to work, and I'm not sure that's how Actions works

Reply:
Yeah, I think it will work because actions are just curated steps that will be performed on the same container.

Reply:
done!